sql-odbc/src/sqlodbc/opensearch_connection.cpp

```diff
@@ -95,6 +95,15 @@ int LIBOPENSEARCH_connect(ConnectionClass *self) {
     rt_opts.crypt.verify_server = (self->connInfo.verify_server == 1);
     rt_opts.crypt.use_ssl = (self->connInfo.use_ssl == 1);
 
+    // Server host url protocol is necessary for sql plugin validation
+    if (rt_opts.conn.server.size() && rt_opts.conn.server.find("http", 0) != 0) {
+        if(rt_opts.crypt.use_ssl) {
+            rt_opts.conn.server = std::string("https://") + std::string(self->connInfo.server);
+        } else {
+            rt_opts.conn.server = std::string("http://") + std::string(self->connInfo.server);
```

Comment:
`rt_opts.conn.server` should follow the same assignment patter as other properties in this method -- `rt_opts.conn.server.assign(...)`